containers/release-candidate/Dockerfile

```diff
@@ -15,20 +15,61 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM registry.fedoraproject.org/fedora-minimal:36
+FROM registry.fedoraproject.org/fedora:36
 
 WORKDIR /root
 
-COPY setup-container.sh /usr/bin/
-RUN /usr/bin/setup-container.sh
+# Delete cruft fedora leaves in the homedir
+RUN rm /root/*
```

Comment:
This command will return an error status if /root contains any visible (not hidden) directories.  If you run `rm -rf /root/*` instead, it will not return any error status.